fix: update generator-client/README.md (#116)
open-rpc/generator-client: README.md

# open-rpc-generator-client

[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=open-rpc/generator-client)](https://dependabot.com) [![Dependabot Status](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/Tf55C7d)

> Multi-language client generator tool for [open-rpc](https://github.com/open-rpc/spec) APIs. Developers generate a client by passing a raw open-rpc schema document to _this_ tool which generates a client based on the supported language templates.

Reviewer: ```suggestion
Multi-language client generator tool for [open-rpc](https://github.com/open-rpc/spec) APIs. Developers generate a client by passing an OpenRPC Document to _this_ tool which generates a client based on the supported language templates.
```

![overview diagram](https://github.com/open-rpc/design/blob/master/diagrams/generator-client/open-rpc-diagrams.png?raw=true)

Supported client languages:

![js badge](https://img.shields.io/badge/js-javascript-yellow.svg) ![rs badge](https://img.shields.io/badge/rs-rust-brown.svg)

## Install

```shell
npm install -g @open-rpc/generator-client
```

Check install or update package

```shell
open-rpc-generator-client --version

npm -g update @open-rpc/generator-client
```

## Usage

### Generating a Client

```shell
open-rpc-generator-client \
  -s https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/petstore-openrpc.json \
  PetStore
```

> Using the `open-rpc-generator-client` command, then passing an example schema document `petstore-openrpc.json` in the directory of `Petstore`.

Reviewer: ```suggestion
Using the `open-rpc-generator-client` command, then passing an example OpenRPC Document `petstore-openrpc.json` in the directory of `Petstore`.
```

### Using Docker ![docker-icon](https://github.com/open-rpc/design/blob/master/png/docker-icon-16x.png?raw=true)

In action (for a js client):

```shell
docker run \
  -v $(pwd)/petstore/:/petstore \

>   openrpc/generator-client-js \

Reviewer: ```suggestion
  openrpc/generator-client \
```
